src/Service/ShipmentService.php

```diff
@@ -300,11 +300,9 @@ private function calculateParcelPriceWithOrderDiscount($order, $currentPrice)
 
     private function calculateProductsPrice($product)
     {
-        if (!empty($product['total_customization_wt'])) {
-            return round($product['total_customization_wt'], 2);
-        }
-
-        return round($product['total_wt'], 2);
+        return $product['total_price_tax_incl'] ?
+            round($product['total_price_tax_incl'], 2) :
+            round($product['total_wt'], 2);
```

Comment:
fyi: In PS.17 there's Number class which does rounding a bit better - this rounding in some cases might be unsafe due to if you have more numbers comma separated it might not produce the expected result.

if YOu use this module for PS16 then you can as well require that number package on your own.

https://packagist.org/packages/prestashop/decimal